docs/home/benefits-of-pwa.md

```diff
@@ -0,0 +1,65 @@
+# Benefits of Progressive Web Apps
+
+Building a Progressive Web App has plenty of advantages, from both a developer and user perspective. If you're looking for a summary of what PWAs can bring to the table, the following article covers several benefits, broken down by how they affect either the development experience or the user experience.
+
+If you are unclear on what a Progressive Web App is from a technical perspective, feel free to check out our [Beginner's Guide to Progressive Web Apps.](/home/pwa-intro/)
+
+## Developer
+
+### Multiple Platforms - Single Codebase
+
+Progressive Web Apps are built with web technology, and as a result, run on any platform with minimal changes to a single, primary codebase. If you develop your app with responsive display design in mind, the same code base will work just as well any mobile (Android, iOS) and any native (Windows, OSX) platform.
+
+You can reach as many users as possible while minimizing the amount of work required to maintain your app, easing development flow and saving time in the process.
+
+### Discoverability
+
+Progressive Web Apps can be packaged and delivered to many native app stores, which allows them to be discoverable from two different paths: on the web and through app stores.
```

Comment:
Potential Edit: Progressive Web Apps are discoverable and installable from both the web and app stores, such as Google Play and the Microsoft Store. And again, with just one codebase.